connexion/api.py

```diff
@@ -40,12 +41,16 @@ def compatibility_layer(spec):
     # Make all response codes be string
     for path_name, methods_available in spec.get('paths', {}).items():
         for method_name, method_def in methods_available.items():
-            if method_name == 'parameters':
+            if method_name == 'parameters' or \
+               not isinstance(method_def, dict):
```

Comment:
This would look nicer with `()` instead of `\`. Also the [new recommendation is to wrap before the operator](http://bugs.python.org/issue26763).
Also a comment linking to #220 would also make this easier to understand in the future.